src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/searchablelist/SearchableList.kt

```diff
@@ -27,12 +27,16 @@ class SearchableList<T>(listItems: ObservableList<T>, outputValue: Property<T>,
         outputValue.bindBidirectional(valueProperty())
         hbox {
             addClass(SearchableListStyles.searchFieldContainer)
-            add(SearchableListStyles.searchIcon().apply { addClass(SearchableListStyles.icon) })
+            add(SearchableListStyles.searchIcon("1.5em").apply { addClass(SearchableListStyles.icon) })
             searchField = JFXTextField()
             searchField.addClass(SearchableListStyles.searchField)
+            searchField.focusColor = Color.TRANSPARENT
             searchField.hgrow = Priority.ALWAYS
             add(searchField)
         }
+        hbox{
+            addClass(SearchableListStyles.separator)
+        }
```

Comment:
Instead of adding another UI element as a spacer, it's probably better to just change the spacing between the field and listview directly by setting the searchable list root Vbox spacing to `30.px`